src/Components/SingleMovie/SingleMovie.js

```diff
@@ -8,8 +8,16 @@ function SingleMovie({movie, goBack, displayVideo}) {
         key={movie.movie.id}
         id={movie.movie.id}
         title={movie.movie.title}
+        tagLine={movie.movie.tagline}
```

Comment:
Instead of passing a bunch of props, consider just passing a `movie` object as a prop and use dot notation to dig into it in the child component